packages/edit-post/src/components/header/index.js

```diff
@@ -189,6 +192,16 @@ function Header( { setEntitiesSavedStatesCallback, initialPost } ) {
 						setEntitiesSavedStatesCallback
 					}
 				/>
+				{ hasActiveMetaboxes && (
+					<Button
+						icon={ metabox }
+						onClick={ () => {
+							toggleMetaBoxes();
+						} }
+						label={ __( 'Show meta boxes' ) }
```

Comment:
As pointed out, I would consider a different label as `meta boxes` might not be a term the general user is familiar with. Also, when you close the meta boxes you would want to have `Hide meta boxes` as the label, so in line with the `Settings` toggle for the sidebar I would consider going with just a noun here.